promoted_python_delivery_client/client/two_arm_experiment.py

```diff
@@ -45,6 +46,22 @@ def check_membership(self, user_id: str) -> Optional[CohortMembership]:
         return None
 
 
+def _do_hash(val: str) -> int:
+    """Generate a stable int hash for a string.
+
+    Currently we're using md5 but have this wrapper to allow for swapping in a faster implementation later.
```

Comment:
Side - we'll have to be careful swapping this later since it'll change which arms users are in.  It's probably still fine to delay the optimization and we can walk any customers through the migration.